### PR TITLE
Adding the requirements document & putting a link to it in the README

### PR DESCRIPTION
**Pull request by l00144427**
Renamed the document to requirements.md & put a link to it in the README.md file

### CODE REVIEW

File: content/requirements.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+* If the username and/or password are incorrect then the user will remain on the home page and an error message will be displayed.
+
+* After logging out the user will return to the home page.
+
+**Data Entry Requirements Details**
+
+* Branch location should be entered.
+
+* Dimensions can be entered.
```
Comment (rlennon) [inline content/requirements.md:45]: Additional information would be useful such as the number of rooms, special room requirements and so on

File: content/requirements.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* The frontend UI should be available during workings hours.
+
+* The frontend UI layout should be simple and clean.
+
+**Outstanding Questions**
+
+Do we need to worry about adding & deleting users?
+
+Do we need to worry about adding & deleting branches?
```
Comment (rlennon) [inline content/requirements.md]: Yes

File: content/requirements.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+**Other Details**
+
+* The frontend UI should be available during workings hours.
+
+* The frontend UI layout should be simple and clean.
+
+**Outstanding Questions**
+
+Do we need to worry about adding & deleting users?
```
Comment (rlennon) [inline content/requirements.md]: yes

**Review by rlennon — CHANGES_REQUESTED** [review]

**Comment by l00144427** [comment]
I added additional information to the Users, Data Entry Requirements Details & Other Details sections and also removed the Outstanding Questions section based on the feedback.
Please review and approve if acceptable.

**Review by cjfarrelly — APPROVED** [review]
lgtm!

**Review by L00144438 — APPROVED** [review]
Need to discuss the requirements!

File: content/requirements.md (hunk 1 above)
Comment (l00144427) [reply content/requirements.md:45]: I have added some changes based on your comments, please review & approve if you think it's ok.

**Review by l00119705 — APPROVED** [review]

**Review by l00073976 — APPROVED** [review]
lgtm

**Review by rlennon — APPROVED** [review]
LGTM

**Pull request merged**